WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -73,7 +73,15 @@ decisions = {
 	support_greens = {
 		only_playable = yes
 		potential = {
-			culture_group = latin
+			OR = {
+				culture_group = latin
+				culture = roman
+				has_landed_title = e_wre
+				has_landed_title = e_roman_empire
+				has_title = c_roma
+				has_title = b_rome
+				has_title b_roma
+			}
```

Comment:
This _only_ applies if a Romance  character has already restored the Hippondrome. Non-Romance characters cannot restore the Hippondrome, but if a non-Romance character takes over when the Hippondrome is fully built, it would be stupid for him not to participate in his realm's most beloved spectacle.

By allowing them to participate if the Hippondrome is already built, but denying them the need to built it, it ensures accuracy on both ends. This is also a very specific circumstance: a person must directly control Rome itself.